great_expectations_cloud/agent/agent.py

```diff
@@ -97,8 +97,7 @@ class GXAgentConfig(AgentBaseExtraForbid):
 
     queue: str
     connection_string: AmqpDsn
-    # pydantic will coerce this string to AnyUrl type
-    gx_cloud_base_url: AnyUrl = CLOUD_DEFAULT_BASE_URL
+    gx_cloud_base_url: AnyUrl = AnyUrl(url=CLOUD_DEFAULT_BASE_URL, scheme="https")
```

Comment:
Is hardcoding the scheme going to make this hard to test locally? If pydantic is going to coerce this, maybe we should let it and put in a type ignore?

Reply:
Why would it make this hard to test? I could move the `AnyUrl` object into the constant. We could let it coerce, but I'm not sure I understand what the reason for doing that would be.

Reply:
The string constant `CLOUD_DEFAULT_BASE_URL` already starts with `https`. `scheme` is just a required field for `AnyUrl`.